pkg/reconciler/dependencybuild/buildrecipeyaml.go

```diff
@@ -33,6 +16,14 @@ spec:
       - name: maven-settings
       - name: source
     params:
+      - name: JAVA_VERSION
```

Comment:
it would make sense to me to add these params after `GOALS` once you add those back

Reply:
This should be fixed in 613a2aa.